### ISSUE TITLE
Third party dependency does not get properly included on packaged component

### ISSUE THREAD

**Issue by inorganik** [dherges/ng-packagr#462]
## Type of Issue

```
[X] Bug Report
[ ] Feature Request
```

## Description

Third party dependency does not get properly included in packaged component.

#### How To Reproduce

I created an angular directive wrapper around my vanilla js lib [Count-Up](https://github.com/inorganik/countUp.js). The unpackaged directive works fine. It imports countup like so: 
```
import * as CountUp from 'countup.js';
```
However when I try to use the component packaged by ng-packagr, it fails.
For ng-packagr options in package.json, I tried this:
```
"ngPackage": {
    "lib": {
      "entryFile": "index.ts",
      "umdModuleIds": {
        "countup.js": "CountUp"
      }
    }
  }
```
As suggested in Nikolas LeBlanc's [article](https://medium.com/@nikolasleblanc/building-an-angular-4-component-library-with-the-angular-cli-and-ng-packagr-53b2ade0701e) I ran `npm pack` on the packaged component, include the resulting `.tgz` file in a new angular project, and npm install the tgz file. 

When I invoke it, I get `CountUp is not a constructor`. With a debugger statement where the vanilla js class is invoked, I can see that after it's packaged, it's putting everything from `CountUp` into `CountUp.default`

![countup-angular2-error](https://user-images.githubusercontent.com/109553/34534220-961f1a86-f07a-11e7-9921-82b73f4f56ed.png)

#### Expected Behaviour

ng-packagr should include the third-party so that it doesn't fail.

#### Version Information

ng-packagr: 1.6.0
angular/cli: 1.6.2
Angular: 4.4.6

**Comment by dherges** [comment]
Hi, can you check the 1.6 README? 1.6 still requires `lib.eternals` to be set.

**Comment by inorganik** [comment]
It appears 1.6 is what you get when you run `npm install ng-packagr`. You probably just need to run `npm publish` again for the latest release.

I checked the 1.6 README and the current README and "lib.eternals", nor just the word "eternal" is in either. :(

I also tried upgrading to the newest release. I cleaned, re-installed everything, made a new .tgz file, and still have the same issue - `CountUp is not a constructor`.
